lib/widgets/tb_elevated_button.dart

```diff
@@ -0,0 +1,87 @@
+import 'package:flutter/material.dart';
+import 'package:tech_byte/utils/colors.dart';
+import 'package:tech_byte/utils/constants.dart';
+
+enum TBButtonType { primary, secondary }
+
+class TBButton extends StatelessWidget {
+  final void Function() onPressed;
+  final String text;
+  final bool enabled;
+  final TBButtonType type;
+  final bool isLoading;
+
+  const TBButton({
+    super.key,
+    required this.onPressed,
+    required this.text,
+    required this.type,
+    this.enabled = true,
+    this.isLoading = false,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return SizedBox(
+      height: kButtonHeight,
+      width: double.infinity,
+      child: ElevatedButton(
+        onPressed: enabled ? onPressed : null,
+        style: ButtonStyle(
+          elevation: type == TBButtonType.primary
+              ? null
+              : const MaterialStatePropertyAll(0),
```

Comment:
By design primary button is not elevated.

Reply:
@milanzezeljgs you just erased elevation, by default ElevatedButton has elevation we want to remove it in both types